### PR TITLE
Euslint test improvement

### PR DESCRIPTION
**Pull request by knorth55**
### What's new

- improve test result output
- add line length test limit to 100
- add white space test
- add indent test (disabled in catkin test now)
- test only `euslisp/*.l`
- fix style `euslisp/*.l`

#1590

### CODE REVIEW

File: jsk_2016_01_baxter_apc/scripts/euslint [hunk 1]
```diff
@@ -45,8 +76,38 @@ def test_parenthesis(result):
...
+
+
+def test_indent(result):
+    if len(result['indent']) != 0:
+        for indent_result in result['indent']:
+            print('WARNING: Incorrect Indent at line {0}: {1} != {2}'
+                  .format(indent_result[0]+1, indent_result[1], indent_result[2]),
+                          file=sys.stderr)
+        return 1
+    return 0
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/scripts/euslint]: Please comment out this.

File: jsk_2016_01_baxter_apc/scripts/euslint [hunk 2]
```diff
@@ -11,31 +11,62 @@ def parse(source_code):
     comment = False
     result = {
         'parenthesis_open': 0,
-        'parenthesis_close':0,
+        'parenthesis_close': 0,
         'tab': [],
+        'line_length': [],
+        'indent': [],
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/scripts/euslint]: comment out

File: jsk_2016_01_baxter_apc/scripts/euslint [hunk 3]
```diff
@@ -11,31 +11,62 @@ def parse(source_code):
...
+        'white_space': [],
     }
+    correct_indent = 0
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/scripts/euslint]: comment out

File: jsk_2016_01_baxter_apc/scripts/euslint [hunk 4]
```diff
@@ -11,31 +11,62 @@ def parse(source_code):
...
+    next_correct_indent = 0
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/scripts/euslint]: comment out

File: jsk_2016_01_baxter_apc/scripts/euslint [hunk 5]
```diff
@@ -11,31 +11,62 @@ def parse(source_code):
...
         elif line.startswith('|#'):
             comment = False
         if comment:
             continue
+        if len(line) > 100:
+            result['line_length'].append([line_num, len(line)])
+        is_beginning = True
+        is_comment = False
+        has_quote = False
+        indent = 0
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/scripts/euslint]: comment out

File: jsk_2016_01_baxter_apc/scripts/euslint [hunk 6]
```diff
@@ -11,31 +11,62 @@ def parse(source_code):
...
+        white_space = 0
+        correct_indent = next_correct_indent
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/scripts/euslint]: comment out

**Comment by knorth55** [comment]
fixed

**Comment by wkentaro** [comment]
Please add `FIXME` on the top of https://github.com/knorth55/jsk_apc/blob/55062e34b0f16b24153f84ef55c53046dfaeee8a/jsk_2016_01_baxter_apc/scripts/euslint .

**Comment by wkentaro** [comment]
+1

**Pull request merged**